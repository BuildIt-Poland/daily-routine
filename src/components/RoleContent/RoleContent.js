// Generic responsive content wrapper

import styled from 'styled-components';

import { spacingMedium } from '../../styles/designTokens';

const Content = styled.div`
  max-width: 48rem;
  padding: 0 ${spacingMedium};
  margin: 0 auto;
  position: relative;
  display: grid;
  grid-template-rows: 1fr auto;
  height: 100%;
  align-items: end;
`;

export default Content;
