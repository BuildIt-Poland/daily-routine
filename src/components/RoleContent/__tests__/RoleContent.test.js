import React from 'react';
import { create } from 'react-test-renderer';

import RoleContent from '../RoleContent';

describe('COMPONENT - RoleContent', () => {
  it('renders correctly', () => {
    const component = create(<RoleContent />);

    expect(component.toJSON()).toMatchSnapshot();
  });
});
