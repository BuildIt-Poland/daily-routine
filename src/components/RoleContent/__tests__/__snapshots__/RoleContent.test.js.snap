// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`COMPONENT - RoleContent renders correctly 1`] = `
.c0 {
  max-width: 48rem;
  padding: 0 2rem;
  margin: 0 auto;
  position: relative;
  display: grid;
  grid-template-rows: 1fr auto;
  height: 100%;
  -webkit-align-items: end;
  -webkit-box-align: end;
  -ms-flex-align: end;
  align-items: end;
}

<div
  className="c0"
/>
`;
